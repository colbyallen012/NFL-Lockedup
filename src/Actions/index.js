export const showCrimes = (crimes) => ({
  type: 'SHOW_CRIMES',
  crimes
})

export const showTeams = (teams) => ({
  type: 'SHOW_TEAMS',
  teams
})

export const showPlayers = (players) => ({
  type: 'SHOW_PLAYERS',
  players
})