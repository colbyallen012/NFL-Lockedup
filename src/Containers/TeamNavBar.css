.nav-bar-container {
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
  justify-content: center;
}

.nav {
  background-color: yellow;
  height: 25px;
  margin: 5px;
  width: 200px;
}