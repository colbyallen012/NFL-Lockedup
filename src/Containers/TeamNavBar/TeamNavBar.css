.nav-bar-container {
  display: flex;
  flex-direction: row;
  flex-wrap: wrap;
  /* justify-content: center; */
}

.nav {
  color: white;
  font-weight: bolder;
  height: 25px;
  margin: 5px auto;
  padding: 5px;
  text-align: center;
  text-decoration: none;
  width: 90%;
}

.nav:focus {
  background-color: #013369;
  border-radius: 20px;
}