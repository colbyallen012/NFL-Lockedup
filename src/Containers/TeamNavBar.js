import React, {Component} from 'react';
import { getOneTeamData } from '../apiCalls/apiCalls'
import { connect } from 'react-redux';
import { showTeamArrests, showCrimes, showPlayers } from '../Actions/index';
import {Route, NavLink} from 'react-router-dom'
import './TeamNavBar.css'
import TeamsContainer from '../Containers/TeamsContainer'
import CrimesContainer from '../Containers/CrimesContainer'

class TeamNavBar extends Component {

  singleTeamArrests = (team) => {
    getOneTeamData(team)
      .then(team => this.props.showTeamArrests(team))
  }

  render () {
    return (
      <div>
        <CrimesContainer crimes={this.props.crimes} players={this.props.players}/>
      <div className='nav-bar-container'>
        <NavLink to='/den' className='nav' onClick={() => this.singleTeamArrests('DEN')}>Denver Broncos</NavLink>
        <NavLink to='/min' className='nav' onClick={() => this.singleTeamArrests('MIN')}>Minnesota Vikings</NavLink>
        <NavLink to='/cin' className='nav' onClick={() => this.singleTeamArrests('CIN')}>Cincinati Bengals</NavLink>   
        <NavLink to='/jac' className='nav' onClick={() => this.singleTeamArrests('JAC')}>Jacksonville Jaguars</NavLink>   
        <NavLink to='/tb' className='nav' onClick={() => this.singleTeamArrests('TB')}>Tampa Bay Buccaneers</NavLink>   
        <NavLink to='/ten' className='nav' onClick={() => this.singleTeamArrests('TEN')}>Tennessee Titans</NavLink>   
        <NavLink to='/cle' className='nav' onClick={() => this.singleTeamArrests('CLE')}>Cleveland Browns</NavLink>   
        <NavLink to='/ind' className='nav' onClick={() => this.singleTeamArrests('IND')}>Indianapolis Colts</NavLink>   
        <NavLink to='/kc' className='nav' onClick={() => this.singleTeamArrests('KC')}>Kansas City Chiefs</NavLink>   
        <NavLink to='/chi' className='nav' onClick={() => this.singleTeamArrests('CHI')}>Chicago Bears</NavLink>   
        <NavLink to='/mia' className='nav' onClick={() => this.singleTeamArrests('MIA')}>Miami Dolphins</NavLink>   
        <NavLink to='/sea' className='nav' onClick={() => this.singleTeamArrests('SEA')}>Seattle Seahawks</NavLink>   
        <NavLink to='/sf' className='nav' onClick={() => this.singleTeamArrests('SF')}>San Francisco 49ers</NavLink>   
        <NavLink to='/bal' className='nav' onClick={() => this.singleTeamArrests('BAL')}>Baltimore Ravens</NavLink>   
        <NavLink to='/lac' className='nav' onClick={() => this.singleTeamArrests('LAC')}>Los Angeles Chargers</NavLink>   
        <NavLink to='/no' className='nav' onClick={() => this.singleTeamArrests('NO')}>New Orleans Saints</NavLink>   
        <NavLink to='/gb' className='nav' onClick={() => this.singleTeamArrests('GB')}>Green Bay Packers</NavLink>   
        <NavLink to='/pit' className='nav' onClick={() => this.singleTeamArrests('PIT')}>Pittsburgh Steelers</NavLink>   
        <NavLink to='/nyj' className='nav' onClick={() => this.singleTeamArrests('NYJ')}>New York Jets</NavLink>   
        <NavLink to='/ari' className='nav' onClick={() => this.singleTeamArrests('ARI')}>Arizona Cardinals</NavLink>   
        <NavLink to='/oak' className='nav' onClick={() => this.singleTeamArrests('OAK')}>Oakland Raiders</NavLink>   
        <NavLink to='/was' className='nav' onClick={() => this.singleTeamArrests('WAS')}>Washington Redskins</NavLink>   
        <NavLink to='/dal' className='nav' onClick={() => this.singleTeamArrests('DAL')}>Dallas Cowboys</NavLink>   
        <NavLink to='/car' className='nav' onClick={() => this.singleTeamArrests('CAR')}>Carolina Panthers</NavLink>   
        <NavLink to='/buf' className='nav' onClick={() => this.singleTeamArrests('BUF')}>Buffalo Bills</NavLink>   
        <NavLink to='/ne' className='nav' onClick={() => this.singleTeamArrests('NE')}>New England Patriots</NavLink>   
        <NavLink to='/atl' className='nav' onClick={() => this.singleTeamArrests('ATL')}>Atlanta Falcons</NavLink>   
        <NavLink to='/phi' className='nav' onClick={() => this.singleTeamArrests('PHI')}>Pjiladelphia Eagles</NavLink>   
        <NavLink to='/nyg' className='nav' onClick={() => this.singleTeamArrests('NYG')}>New York Giants</NavLink>   
        <NavLink to='/la' className='nav' onClick={() => this.singleTeamArrests('LA')}>Los Angeles Rams</NavLink>   
        <NavLink to='/det' className='nav' onClick={() => this.singleTeamArrests('DET')}>Detroit Lions</NavLink> 
        <NavLink to='/hou' className='nav' onClick={() => this.singleTeamArrests('HOU')}>Houston Texans</NavLink>
        </div>
        <Route exact path='/Den' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Min' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Cin' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Jac' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Tb' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Ten' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Cle' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Ind' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Kc' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Chi' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Mia' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Sea' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Sf' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Bal' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Lac' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/No' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Gb' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Pit' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Nyj' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Ari' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Oak' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Was' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Dal' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Car' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Buf' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Ne' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Atl' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Phi' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Nyg' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/La' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
        <Route exact path='/Det' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>
         <Route exact path='/Hou' render={() => 
            <TeamsContainer arrests={this.props.teamArrests}/> 
        }/>            
      </div>
    )
  }
}

const mapStateToProps = (store) => ({
  crimes: store.crimes,
  players: store.players,
  teamArrests: store.teamArrests
})

const mapDispatchToProps = dispatch => ({
  showCrimes: (crimes) => dispatch(showCrimes(crimes)),
  showPlayers: (players) => dispatch(showPlayers(players)),
  showTeamArrests: (team) => dispatch(showTeamArrests(team))
})

export default connect(mapStateToProps, mapDispatchToProps)(TeamNavBar);