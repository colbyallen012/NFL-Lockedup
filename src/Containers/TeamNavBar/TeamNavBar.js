import React, {Component} from 'react';
import { getOneTeamData } from '../../apiCalls/apiCalls'
import { connect } from 'react-redux';
import { showTeamArrests, showCrimes, showPlayers, emptyState } from '../../Actions/index';
import { NavLink } from 'react-router-dom'
import './TeamNavBar.css'

export class TeamNavBar extends Component {

  singleTeamArrests = (team) => {
    getOneTeamData(team)
      .then(team => this.props.showTeamArrests(team))
  }

  emptyState = () => {
    this.props.emptyState()
  }

  render () {
    return (
      <div>
      <div className='nav-bar-container'>
        <NavLink to='/' className='nav' onClick={() => this.emptyState()}>Home</NavLink>
        <NavLink to='/den' className='nav' onClick={() => this.singleTeamArrests('DEN')}>Denver Broncos</NavLink>
        <NavLink to='/min' className='nav' onClick={() => this.singleTeamArrests('MIN')}>Minnesota Vikings</NavLink>
        <NavLink to='/cin' className='nav' onClick={() => this.singleTeamArrests('CIN')}>Cincinati Bengals</NavLink>   
        <NavLink to='/jac' className='nav' onClick={() => this.singleTeamArrests('JAC')}>Jacksonville Jaguars</NavLink>   
        <NavLink to='/tb' className='nav' onClick={() => this.singleTeamArrests('TB')}>Tampa Bay Buccaneers</NavLink>   
        <NavLink to='/ten' className='nav' onClick={() => this.singleTeamArrests('TEN')}>Tennessee Titans</NavLink>   
        <NavLink to='/cle' className='nav' onClick={() => this.singleTeamArrests('CLE')}>Cleveland Browns</NavLink>   
        <NavLink to='/ind' className='nav' onClick={() => this.singleTeamArrests('IND')}>Indianapolis Colts</NavLink>   
        <NavLink to='/kc' className='nav' onClick={() => this.singleTeamArrests('KC')}>Kansas City Chiefs</NavLink>   
        <NavLink to='/chi' className='nav' onClick={() => this.singleTeamArrests('CHI')}>Chicago Bears</NavLink>   
        <NavLink to='/mia' className='nav' onClick={() => this.singleTeamArrests('MIA')}>Miami Dolphins</NavLink>   
        <NavLink to='/sea' className='nav' onClick={() => this.singleTeamArrests('SEA')}>Seattle Seahawks</NavLink>   
        <NavLink to='/sf' className='nav' onClick={() => this.singleTeamArrests('SF')}>San Francisco 49ers</NavLink>   
        <NavLink to='/bal' className='nav' onClick={() => this.singleTeamArrests('BAL')}>Baltimore Ravens</NavLink>   
        <NavLink to='/lac' className='nav' onClick={() => this.singleTeamArrests('LAC')}>Los Angeles Chargers</NavLink>   
        <NavLink to='/no' className='nav' onClick={() => this.singleTeamArrests('NO')}>New Orleans Saints</NavLink>   
        <NavLink to='/gb' className='nav' onClick={() => this.singleTeamArrests('GB')}>Green Bay Packers</NavLink>   
        <NavLink to='/pit' className='nav' onClick={() => this.singleTeamArrests('PIT')}>Pittsburgh Steelers</NavLink>   
        <NavLink to='/nyj' className='nav' onClick={() => this.singleTeamArrests('NYJ')}>New York Jets</NavLink>   
        <NavLink to='/ari' className='nav' onClick={() => this.singleTeamArrests('ARI')}>Arizona Cardinals</NavLink>   
        <NavLink to='/oak' className='nav' onClick={() => this.singleTeamArrests('OAK')}>Oakland Raiders</NavLink>   
        <NavLink to='/was' className='nav' onClick={() => this.singleTeamArrests('WAS')}>Washington Redskins</NavLink>   
        <NavLink to='/dal' className='nav' onClick={() => this.singleTeamArrests('DAL')}>Dallas Cowboys</NavLink>   
        <NavLink to='/car' className='nav' onClick={() => this.singleTeamArrests('CAR')}>Carolina Panthers</NavLink>   
        <NavLink to='/buf' className='nav' onClick={() => this.singleTeamArrests('BUF')}>Buffalo Bills</NavLink>   
        <NavLink to='/ne' className='nav' onClick={() => this.singleTeamArrests('NE')}>New England Patriots</NavLink>   
        <NavLink to='/atl' className='nav' onClick={() => this.singleTeamArrests('ATL')}>Atlanta Falcons</NavLink>   
        <NavLink to='/phi' className='nav' onClick={() => this.singleTeamArrests('PHI')}>Philadelphia Eagles</NavLink>   
        <NavLink to='/nyg' className='nav' onClick={() => this.singleTeamArrests('NYG')}>New York Giants</NavLink>   
        <NavLink to='/la' className='nav' onClick={() => this.singleTeamArrests('LA')}>Los Angeles Rams</NavLink>   
        <NavLink to='/det' className='nav' onClick={() => this.singleTeamArrests('DET')}>Detroit Lions</NavLink> 
        <NavLink to='/hou' className='nav' onClick={() => this.singleTeamArrests('HOU')}>Houston Texans</NavLink>
      </div>       
      </div>
    )
  }
}

export const mapStateToProps = (store) => ({
  crimes: store.crimes,
  players: store.players,
  teamArrests: store.teamArrests
})

export const mapDispatchToProps = dispatch => ({
  showCrimes: (crimes) => dispatch(showCrimes(crimes)),
  showPlayers: (players) => dispatch(showPlayers(players)),
  showTeamArrests: (team) => dispatch(showTeamArrests(team)),
  emptyState: () => dispatch(emptyState())
})

export default connect(mapStateToProps, mapDispatchToProps)(TeamNavBar);