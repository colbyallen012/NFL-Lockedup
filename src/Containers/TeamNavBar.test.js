import React from 'react';
import { shallow } from 'enzyme';
import { TeamNavBar, mapStateToProps, mapDispatchToProps } from './TeamNavBar';
import { showTeamArrests, showCrimes, showPlayers, emptyState } from '../Actions';

describe('TeamNavBar', () => {
  let wrapper;
  let mockEmptyState;
  let mockShowCrimes;
  let mockShowPlayers;
  let mockShowTeamArrests;

  const mockCrimes = [{
    Category: "DUI",
    arrest_count: "224"
    },
    {
    Category: "Drugs",
    arrest_count: "107"
  }]

  const mockPlayers = [{
    Name: "Kenny Britt",
    Team: "TEN",
    Team_name: "Titans",
    Team_city: "Nashville",
    Position: "WR",
    arrest_count: "7"
  }]

  const mockTeamArrests = [{
    arrest_stats_id: "903",
    Date: "2018-10-22",
    Team: "DEN",
    Team_name: "Broncos",
    Team_preffered_name: "Denver Broncos",
    Team_city: "Denver",
    Team_logo_id: "10",
    Team_Conference: "AFC",
    Team_Division: "West",
    Team_Conference_Division: "AFC West",
    Team_hex_color: "FC4C02",
    Team_hex_alt_color: "0C2340",
    reddit_group_id: "3",
    Name: "Chad Kelly",
    Position: "QB",
    Position_name: "Quarterback",
    Position_type: "O",
    Encounter: "Arrested",
    Category: "Trespassing",
    Crime_category: "Other",
    Crime_category_color: "BCBD22",
    Description: "The Englewood Colorado Police arrested and charged Kelly with first-degree criminal trespassing after a report of a man allegedly inside a home.",
    Outcome: "Resolution Undetermined.",
    Season: "2018",
    ArrestSeasonState: "InSeason",
    overall_category_id: "15",
    general_category_id: "15",
    legal_level_id: "1",
    resolution_category_id: "1",
    Year: "2018",
    Month: "10",
    Day: "22",
    Day_of_Week: "Monday",
    Day_of_Week_int: "2",
    YearToDate: "0",
    DaysSince: "244",
    DaysToLastArrest: "36",
    DaysToLastCrimeArrest: null,
    DaysToLastTeamArrest: "229"
  }]

  beforeEach(() => {
    mockEmptyState = jest.fn()
    mockShowCrimes = jest.fn()
    mockShowPlayers = jest.fn()
    mockShowTeamArrests = jest.fn()
    
    wrapper = shallow(
      <TeamNavBar
        crimes={mockCrimes}
        players={mockPlayers}
        teamArrests={mockTeamArrests}
      />
    )
  });

  it('should match snapshot',() => {
    expect(wrapper).toMatchSnapshot() 
  });

  describe('mapStateToProps', () => {
    it('should return arrays with the crimes, players, and arrests', () => {
      const mockState = {
        crimes: [{mockCrimes}],
        players: [{mockPlayers}],
        teamArrests: [{mockTeamArrests}]
      }
      const expected ={
        crimes: [{mockCrimes}],
        players: [{mockPlayers}],
        teamArrests: [{mockTeamArrests}]
      }
      const mappedProps = mapStateToProps(mockState)
  
      expect(mappedProps).toEqual(expected)
    });
  });

  describe('mapdispatchToProps', () => {
    it('calls dispatch with showCrimes action', () => {
      const mockDispatch = jest.fn();
      const actionToDispatch = showCrimes([{category: 'DUI', arrests: 20}]);
      const mappedProps = mapDispatchToProps(mockDispatch);

    mappedProps.showCrimes([{category: 'DUI', arrests: 20}])

    expect(mockDispatch).toHaveBeenCalledWith(actionToDispatch)
    })
  })

})