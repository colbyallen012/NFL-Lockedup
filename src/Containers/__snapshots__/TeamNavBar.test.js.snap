// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TeamNavBar should match snapshot 1`] = `
<div>
  <div
    className="nav-bar-container"
  >
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/"
    >
      Home
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/den"
    >
      Denver Broncos
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/min"
    >
      Minnesota Vikings
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/cin"
    >
      Cincinati Bengals
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/jac"
    >
      Jacksonville Jaguars
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/tb"
    >
      Tampa Bay Buccaneers
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/ten"
    >
      Tennessee Titans
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/cle"
    >
      Cleveland Browns
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/ind"
    >
      Indianapolis Colts
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/kc"
    >
      Kansas City Chiefs
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/chi"
    >
      Chicago Bears
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/mia"
    >
      Miami Dolphins
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/sea"
    >
      Seattle Seahawks
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/sf"
    >
      San Francisco 49ers
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/bal"
    >
      Baltimore Ravens
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/lac"
    >
      Los Angeles Chargers
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/no"
    >
      New Orleans Saints
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/gb"
    >
      Green Bay Packers
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/pit"
    >
      Pittsburgh Steelers
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/nyj"
    >
      New York Jets
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/ari"
    >
      Arizona Cardinals
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/oak"
    >
      Oakland Raiders
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/was"
    >
      Washington Redskins
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/dal"
    >
      Dallas Cowboys
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/car"
    >
      Carolina Panthers
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/buf"
    >
      Buffalo Bills
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/ne"
    >
      New England Patriots
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/atl"
    >
      Atlanta Falcons
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/phi"
    >
      Pjiladelphia Eagles
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/nyg"
    >
      New York Giants
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/la"
    >
      Los Angeles Rams
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/det"
    >
      Detroit Lions
    </NavLink>
    <NavLink
      className="nav"
      onClick={[Function]}
      to="/hou"
    >
      Houston Texans
    </NavLink>
  </div>
  <div>
    <Route
      exact={true}
      path="/Min"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Cin"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Jac"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Tb"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Ten"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Cle"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Ind"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Kc"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Chi"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Mia"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Sea"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Sf"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Bal"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Lac"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/No"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Gb"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Pit"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Nyj"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Ari"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Oak"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Was"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Dal"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Car"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Buf"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Ne"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Atl"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Phi"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Nyg"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/La"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Det"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/Hou"
      render={[Function]}
    />
  </div>
</div>
`;
